spec/hbw/features/bp_form/description_string_spec.rb

```diff
@@ -0,0 +1,21 @@
+feature 'Check description on top and botton of string', js: true do
+  before(:each) do
+    order_type = FactoryBot.create(:order_type, :support_request)
+    FactoryBot.create(:order, order_type: order_type).update(code: 'ORD-35')
+    user = FactoryBot.create(:user)
+
+    signin(user.email, user.password)
+    expect(page).not_to have_content 'Sign in'
+    expect(page).to     have_content 'Orders list'
+
+    click_and_wait 'ORD-35'
+  end
+
+  scenario 'should contain a top description string' do
+    expect(page).to have_selector "[name='topDescriptionString']"
+    expect(page).to have_content 'Top description test'
+
+    expect(page).to have_selector "[name='bottomDescriptionString']"
+    expect(page).to have_content 'Bottom description test'
```

Comment:
You are saying `should contain a top description` but checking top and bottom.
I agree that we can check both places on the same form but let's do some specified checks like that the bottom description text is connected to the proper input.